lib/helpers/normalizeArgs.js

```diff
@@ -38,7 +36,7 @@ function similarTasks(registry, queryTask) {
   var tasks = registry.tasks();
   var similarTasks = [];
   for (var task in tasks) {
-    if (tasks.hasOwnProperty(task)) {
+    if (Object.prototype.hasOwnProperty.call(tasks, task)) {
```

Comment:
@sttk We need to figure out how to get the coverage at 100% around this function.

Reply:
@phated I've added a test case to fix this.

Reply:
Excellent. Thank you! 🙏